src/Controller/Admin/MentoringController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Mentoring;
+use App\Repository\MentoringRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/admin", name="admin_")
+ */
+
+class MentoringController extends AbstractController
+{
```

Comment:
Not sure it's naming it Mentoring controller is a good idea because we already have one in another section. To discuss with the team.